plugin.video.filmfriend.de/resources/lib/jsonparser.py

```diff
@@ -89,6 +89,18 @@ def fetchJson(url,headers=None):
 def parseMain():
 	j = requests.get(f'{base}tenant-groups/21960588-0518-4dd3-89e5-f25ba5bf5631/navigation').json()
 
+def parseGenres():
+	# get list of genres
+	tenant = lm4utils.getSetting("tenant")
+	response_genres = requests.get(f'https://api.tenant.frontend.vod.filmwerte.de/v11/{tenant}/genres')
+	genre_names = []
+	genre_ids = []
+	if response_genres.status_code == 200:
+		genres = response_genres.json()
+		genre_names = [d["name"] for d in genres]
+		genre_ids = [d["id"] for d in genres]
```

Comment:
```suggestion
	genres = fetchJson(f'https://api.tenant.frontend.vod.filmwerte.de/v11/{tenant}/genres')
	genre_names = [d["name"] for d in genres]
	genre_ids = [d["id"] for d in genres]
```

What do you think about using the `fetchJson` method like this?
It encapsulates the `AcceptLanguage`  header plus status code handling. Especially in case of a non-success it gives a little hint to the user instead of silently doing nothing.